google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesWizardAction.java

```diff
@@ -61,13 +64,31 @@ public void actionPerformed(AnActionEvent e) {
   /** The wizard for the "Add Cloud Libraries" menu action. */
   private static final class AddCloudLibrariesWizard extends AbstractWizard<Step> {
 
+    private final SelectClientLibrariesStep selectClientLibrariesStep;
+    private final ManageCloudApisStep manageCloudApisStep;
+
     AddCloudLibrariesWizard(Project project) {
       super(GctBundle.message("cloud.libraries.dialog.title"), project);
-      addStep(new SelectClientLibrariesStep());
-      addStep(new ManageCloudApisStep());
+
+      selectClientLibrariesStep = new SelectClientLibrariesStep(project);
+      selectClientLibrariesStep.addModuleSelectionListener(event -> updateButtons());
+      manageCloudApisStep = new ManageCloudApisStep();
+
+      addStep(selectClientLibrariesStep);
+      addStep(manageCloudApisStep);
       init();
     }
 
+    /** Returns the set of selected {@link CloudLibrary CloudLibraries}. */
+    Set<CloudLibrary> getSelectedLibraries() {
+      return selectClientLibrariesStep.getSelectedLibraries();
+    }
+
+    @Override
+    protected boolean canGoNext() {
+      return selectClientLibrariesStep.getSelectedModule() != null;
```

Comment:
what are you thoughts on adding warning messages in this PR? I'm ok with a separate PR if you think its best.

Reply:
@patflynn and I talked a bit about this this morning; we could either warn/error the user somehow to let them know they have to select a module, or we could default to the "top-most" module, which is rather easy. That way, a module would always be selected and we could remove this disabling logic.

WDYT?

Reply:
defaulting to the top module sgtm. this is really only something we could know for sure from a user study. but having a default should cover most cases imo

Reply:
Done.